src/visitors/inline_visitor.cpp

```diff
@@ -318,12 +310,33 @@ void InlineVisitor::visit_wrapped_expression(WrappedExpression& node) {
 }
 
 void InlineVisitor::visit_program(Program& node) {
+    if (!is_inlineable(node)) {
+        return;
+    }
+
     program_symtab = node.get_symbol_table();
     if (program_symtab == nullptr) {
         throw std::runtime_error("Program node doesn't have symbol table");
     }
+
     node.visit_children(*this);
 }
 
+class CanInlineVisitor: public ConstAstVisitor {
+  private:
+    void visit_verbatim(const ast::Verbatim&) override {
+        is_inlineable = false;
+    }
+
+  public:
+    bool is_inlineable = true;
+};
```

Comment:
Just want to point out that we also have https://github.com/BlueBrain/nmodl/blob/164ed7ebb15b281c2b1380d4b5fb906ea2cac346/src/visitors/inline_visitor.cpp#L24 

maybe good to clarify when one is supposed to be used over other?

Reply:
I implemented this to block it from inlining `SOLVE proc`. However, that inlining happens very differently. Therefore, this is somewhat useless. I can't quite make up my mind if not allowing inlining anything with a VERBATIM block is a good idea; or if it's needless pessimization.

Reply:
Until someone does:
```
#define return_ return
```
and it all breaks.